op-batcher/flags/flags.go

```diff
@@ -101,39 +105,11 @@ var (
 		Required: true,
 		EnvVar:   prefixEnvVar("SEQUENCER_BATCH_INBOX_ADDRESS"),
 	}
+)
 
-	/* Optional Flags */
+func init() {
 
```

Comment:
Is this meant to be an empty `init` function?

Reply:
Removed.